### ISSUE TITLE
Calculated path is not Accurate

### ISSUE THREAD

**Issue by abr1001** [Azure-Samples/AzureMapsCodeSamples#91]
![1234](https://user-images.githubusercontent.com/111348433/192286783-59986700-2c77-4579-b697-154b19444e9c.PNG)

Hello, 
I have question, Min Distance between 2 points(blue markers) are shown with black line. but actual minimum distance is another line(Yellow highlighted). How can i figure out this problem?

Thanks

**Comment by rbrundritt** [comment]
Is this in regard to one of the samples? If not, please post your question in the developer forums and provide more details (your query, how are your black and blue lines being generated?): https://learn.microsoft.com/en-us/answers/topics/azure-maps.html

**Comment by abr1001** [comment]
I make my own network(which is shown with blue), while black lines are the calculated shortest distance across that network.

**Comment by abr1001** [comment]
yes, I am running GeoJSON routing sample on my own machine,
Now i have uploaded my network on your online sample(for your understanding). You can see results are same 
![12345](https://user-images.githubusercontent.com/111348433/192304327-e1ce686d-de0f-45cd-b0c8-d6577988ee9e.PNG)

**Comment by rbrundritt** [comment]
Can you post your GeoJSON network file.

**Comment by abr1001** [comment]
[Eupipe.zip](https://github.com/Azure-Samples/AzureMapsCodeSamples/files/9647322/Eupipe.zip)
here is my network, you can zoom in between Switzerland and Belgium to find my area of interest.
![21](https://user-images.githubusercontent.com/111348433/192310299-62cb7845-7567-427c-acd8-b68181e01664.PNG)

**Comment by rbrundritt** [comment]
I took a look and was able to reproduce what you are seeing. After trying a lot of stuff I found that your lines cross but don't have a node at the intersection. This is what is causing the disconnect as the lines cross but aren't connected. This is equivalent to a bridge over top of another road, thus they don't connect. You will need to find all the intersection points of your lines and break them into two at that point to create nodes. This will turn your data into a true network graph.
